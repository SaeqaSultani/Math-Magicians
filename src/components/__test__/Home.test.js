import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Home from '../Home';

it('Check for any changes to the component', () => {
  const tree = renderer.create(
    <Home />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

it('A heading element should be rendered', () => {
  render(
    <Home />,
  );
  const h1 = screen.getByRole('heading');
  expect(h1).toBeInTheDocument();
});
