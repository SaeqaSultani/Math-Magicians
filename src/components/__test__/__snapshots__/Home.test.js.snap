// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check for any changes to the component 1`] = `
<div
  className="div-container"
>
  <h2>
    Welcome to our page!
  </h2>
  <p
    className="text"
  >
    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages.
  </p>
  <p
    className="text"
  >
    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages.
  </p>
</div>
`;
